Neuro/Vision/Patches/ExileController_Patches.cs

```diff
@@ -9,5 +9,6 @@ public static class ExileController_WrapUp
     public static void Postfix()
     {
         NeuroPlugin.Instance.Vision.ResetAfterMeeting();
+        NeuroPlugin.Instance.Impostor.ResetAfterMeeting();
```

Comment:
Out of scope call to ImpostorHandler. Consider moving this to a new patch in Impostor/Patches, or maybe think of a different solution on how this can be implemented

Reply:
This is from code used in stubs anyway, so I’ll probably just remove it